src/test/java/org/sagebionetworks/bridge/validators/ActivityEventValidatorTest.java

```diff
@@ -36,12 +38,14 @@ public void validates() {
         Validate.entityThrowingException(INSTANCE, getEvent().build());
     }
     
-    @Test
+    @Test(expectedExceptions = IllegalStateException.class,
+            expectedExceptionsMessageRegExp = ".*No update type configured.*")
     public void eventIdNull() {
         ActivityEvent event = getEvent()
                 .withObjectType(null)
                 .withObjectId(null)
                 .withEventType(null)
+                .withUpdateType(null)
                 .withAnswerValue(null)
                 .build();
         assertValidatorMessage(INSTANCE, event, "eventId", EVENT_ID_ERROR);
```

Comment:
This test originally tested if the eventId is null. Is this still possible? If so, the test should test that scenario.

Reply:
Yes you're right, this is a separate test. Added a test for a null ID back to the tests.